File: SAL_VM/SourceCode/SAL_ZX21_VM_UI/SAL_ZX21_GDU.vb
Imports SAL_VM.STACK_VM.ZX81_GPU

Public Class SAL_ZX21_GDU

    ''' <summary>
    '''fd – forward
    '''bk – backward
    '''rt – right
    '''lt – left
    '''cs – clearscreen
    '''pu − penup
    '''pd − pendown
    '''ht − hideturtle
    '''st − showturtle
    ''' </summary>
    ''' <param name="Cmd"></param>
    Public Sub Decode(ByRef Cmd As LogoCmd)
        Me.GDU_DISPLAY.Decode(Cmd)
    End Sub




End Class

File: SpydazWeb AI Virtual Machine/SourceCode/SAL_ZX21_VM_UI/SAL_ZX21_GDU.cs

namespace SAL_VM
{
    public partial class SAL_ZX21_GDU
    {
        public SAL_ZX21_GDU()
        {
            InitializeComponent();
        }

        /// <summary>
    /// fd – forward
    /// bk – backward
    /// rt – right
    /// lt – left
    /// cs – clearscreen
    /// pu − penup
    /// pd − pendown
    /// ht − hideturtle
    /// st − showturtle
    /// </summary>
    /// <param name="Cmd"></param>
        public void Decode(ref STACK_VM.ZX81_GPU.LogoCmd Cmd)
        {
            GDU_DISPLAY.Decode(ref Cmd);
        }
    }
}